CMakeLists.txt

```diff
@@ -308,10 +326,27 @@ if(BUILD_SHARED_LIBS)
         set_target_properties(${LIBRARY_NAME} PROPERTIES VERSION "${VERSION_STRING}")
         set_target_properties(${LIBRARY_NAME} PROPERTIES SOVERSION "${VERSION_MAJOR}")
     endif()
+else()
+    message(STATUS "Linking against dependent libraries statically")
+endif()
 
-    # On Win32, build StormLib.dll since we don't want to clash with Storm.dll
-    if(WIN32)
-        set_target_properties(${LIBRARY_NAME} PROPERTIES OUTPUT_NAME StormLib)
+if(WIN32)
+    if (UNICODE)
+        if (BUILD_SHARED_LIBS)
+            set_target_properties(${LIBRARY_NAME} PROPERTIES OUTPUT_NAME_DEBUG StormLibDUD)
+            set_target_properties(${LIBRARY_NAME} PROPERTIES OUTPUT_NAME_RELEASE StormLibRUD)
+        else()
+            set_target_properties(${LIBRARY_NAME} PROPERTIES OUTPUT_NAME_DEBUG StormLibDUS)
```

Comment:
See previous comment, this is not correct as cmake will link with /MD by default.